templates/testpmd/resource-specs/sriov-vm1-template.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: dpdk-vm-template
+objects:
+- apiVersion: kubevirt.io/v1
+  kind: VirtualMachine
+  metadata:
+    finalizers:
+    - k8s.v1.cni.cncf.io/kubeMacPool
+    generation: 5
+    labels:
+      kubevirt.io/vm: ${VM_NAME}
+    name: ${VM_NAME}
+    namespace: sriov-dpdk-ns
+  spec:
+    running: true
+    template:
+      metadata:
+        labels:
+          kubevirt.io/domain: ${VM_NAME}
+          kubevirt.io/vm: ${VM_NAME}
+      spec:
+        domain:
+          cpu:
+            cores: ${{NUM_OF_CORES}}
+          devices:
+            disks:
+            - disk:
+                bus: virtio
+              name: containerdisk
+            - disk:
+                bus: virtio
+              name: cloudinitdisk
+            interfaces:
+            - masquerade: {}
+              name: default
+              macAddress: ${PRIMARY_MAC}
+            - macAddress: ${SECONDARY_MAC}
+              name: sriov-test-network
+              sriov: {}
+            rng: {}
+          machine:
+            type: pc-q35-rhel8.4.0
+          resources:
+            requests:
```

Comment:
do we need to expose `hugepages`?
```
memory:
            guest: ${MEMORY}
            hugepages:
              pageSize: ${HUGEPAGES_PAGE_SIZE}
```

Reply:
I would expose it as well since hugepages size should be configured relatively to the VM memory.

Reply:
I agree that this should be exposed